code/modules/assembly/bomb.dm

```diff
@@ -42,7 +42,7 @@
 
 		qdel(src)
 		return
-	if((istype(W, /obj/item/weldingtool) && W:welding))
+	if(istype(W, /obj/item/weldingtool) && W:welding)

```

Comment:
`W:welding`. 

😠

Reply:
I cringed when I saw how many colons were in this